### PR TITLE
fix: cw2929 undefined method 'zero?' for nil:NilClass

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

Fixes https://linear.app/chatwoot/issue/CW-2929/nomethoderror-undefined-method-zero-for-nilnilclass-nomethoderror

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by scmmishra — APPROVED** [review]

File: app/services/action_service.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def remove_label(labels)
   end
 
   def assign_team(team_ids = [])
-    return unassign_team if team_ids[0].zero?
+    return unassign_team if team_ids[0]&.zero?
```
Comment (scmmishra) [inline app/services/action_service.rb:52]: Should we use `team_ids.empty?` instead?

Comment (pranavrajs) [reply app/services/action_service.rb:52]: This is better. @vishnu-narayanan Please add a spec to cover the case.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Minor comments.

File: app/services/action_service.rb (hunk 1 above)
Comment (vishnu-narayanan) [reply app/services/action_service.rb:52]: done

**Comment by vishnu-narayanan** [comment]
@pranavrajs If team_id is `nil`, is it safe to assume the team is being unassigned and process accordingly?

File: app/services/action_service.rb (hunk 1 above)
Comment (pranavrajs) [reply app/services/action_service.rb:52]: @vishnu-narayanan I guess you missed @scmmishra's comment.

**Comment by pranavrajs** [comment]
> If team_id is nil, is it safe to assume the team is being unassigned and process accordingly?

@vishnu-narayanan Yes, let us keep this flow for now. We can introduce a custom un-assign team option if needed.

**Comment by vishnu-narayanan** [comment]
@pranavrajs Thank you confirming. Could you please approve if this is good to merge?

**Review by scmmishra — APPROVED** [review]

**Pull request merged**